asn1crypto/cms.py

```diff
@@ -729,6 +729,10 @@ class RecipientKeyIdentifier(Sequence):
         ('other', OtherKeyAttribute, {'optional': True}),
     ]
 
+    def _setup(self):
+        super(RecipientKeyIdentifier, self)._setup()
```

Comment:
`super()._setup()` should be fine

Reply:
This project targets Python 2.6+ currently, hence the parameters passed to super.

Reply:
Unexpected as 2.6 has been EOL for 9, 2.7 for 3 years, but you'll have your reasons.

Reply:
Yeah, I’m more interested in tests for the code you added